src/extension/webviews/query_page/App.tsx

```diff
@@ -288,6 +293,7 @@ export const App: React.FC = () => {
       )}
       {error && <Error multiline={error.includes('\n')}>{error}</Error>}
       {warning && <Warning>{warning}</Warning>}
+      <Tooltip>{stats}</Tooltip>
```

Comment:
I would probably give it its styling to avoid unintended changes to one or the other. Also it probably doesn't need the slightly rounded corners.

It should probably not show if there are no stats, like the warning bar. It currently shows up as an empty narrow bar when just fetching the SQL.